scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/ConvertToNewScala3Syntax.scala

```diff
@@ -18,6 +15,7 @@ object ConvertToNewScala3Syntax extends FormatTokensRewrite.RuleFactory {
   override def create(ftoks: FormatTokens): FormatTokensRewrite.Rule =
     new ConvertToNewScala3Syntax(ftoks)
 
+  private def isSimpleRepeated(t: Term.Repeated): Boolean = t.expr.is[Term.Name]
```

Comment:
```suggestion
  private def isSimpleRepeated(t: Term.Repeated): Boolean = t.expr.isNot[Term.ApplyInfix]
```
?

Reply:
thank you, applied -- and added the check for `isEnclosedInParens` as well.